komodo/symlink/create_links.py

```diff
@@ -17,6 +17,13 @@ def working_dir(path):
         os.chdir(prev_dir)
 
 
+def traverse_symlink_tree(key, link_dict):
```

Comment:
This should be renamed to something that describes what it is doing (locating implicitly moved symlinks), not how it is doing it.

Reply:
Yes, agreed